packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -295,6 +296,22 @@ export interface FieldProps<TCustomMetadata = unknown> {
 	readonly metadata?: FieldSchemaMetadata<TCustomMetadata>;
 }
 
+/**
+ * Additional information to provide to a {@link FieldSchema}. Includes fields for alpha features.
+ *
+ * @typeParam TCustomMetadata - Custom metadata properties to associate with the field.
+ * See {@link FieldSchemaMetadata.custom}.
+ *
+ * @alpha
```

Comment:
I believe the new input tag would apply here.
```suggestion
 * @alpha @input
```